### ISSUE TITLE
[BUG] Incorrect datetimes returned for specific tag

### ISSUE THREAD

**Issue by robertmcleod2** [rtdip/core#825]
**13/09/2024: Removed all sensitive data, and replaced with `redacted`. Please reach out to me for the relevant information.**

**Describe the bug**

When retrieving data for a specific tag with the resample method, the incorrect datetime range is being returned. This is only occuring with the tag: 'redacted'. It returns an additional datetime row incorrectly. When requesting data for other tags, this additional datetime row is not returned. 

**To Reproduce**

```python
from rtdip_sdk.queries import resample
from rtdip_sdk.connectors.odbc.db_sql_connector import DatabricksSQLConnection
from dotenv import load_dotenv
import os

load_dotenv()

SSIP_SERVER_HOSTNAME = os.getenv("SSIP_SERVER_HOSTNAME")
SSIP_HTTP_PATH = os.getenv("SSIP_HTTP_PATH")
SSIP_PAT_TOKEN = os.getenv("SSIP_PAT_TOKEN")

conn = DatabricksSQLConnection(
    server_hostname=SSIP_SERVER_HOSTNAME,
    http_path=SSIP_HTTP_PATH,
    access_token=SSIP_PAT_TOKEN,
)
tag_selection = ['redacted']

template_dict = {
    "business_unit": "`redacted`",
    "region": "`redacted`",
    "asset": "`redacted`",
    "data_security_level": "`redacted`",
    "include_bad_data": False,
    "time_interval_rate": "1",
    "time_interval_unit": "minute",
    "agg_method": "avg",
    "data_type": "float",
    "tag_names": tag_selection,
    "start_date": "2024-01-23T00:28:00",
    "end_date": "2024-01-23T00:32:00",
    "pivot": True,
}

ssip_data = resample.get(conn, template_dict)
print(ssip_data)
```
it is returning the below table with an extra row of data:

| EventTime | `redacted` |
| -- | -- |
| 2024-01-23 00:28:00+00:00 | `redacted` |
| 2024-01-23 00:29:00+00:00 | `redacted` |
| 2024-01-23 00:30:00+00:00 | `redacted` |
| 2024-01-23 00:31:00+00:00 | `redacted` |
| 2024-01-23 00:32:00+00:00 | `redacted` |

**Expected behavior**

It should return the below table:

| EventTime | `redacted` |
| -- | -- |
| 2024-01-23 00:28:00+00:00 | `redacted` |
| 2024-01-23 00:29:00+00:00 | `redacted` |
| 2024-01-23 00:30:00+00:00 | `redacted` |
| 2024-01-23 00:31:00+00:00 | `redacted` |

**Installation Setup (please complete the following information):**

 - OS: tested in both windows and linux
 - Python Version: 3.11
 - SDK Version: 0.12.2

**Comment by robertmcleod2** [comment]
I dont think this issue has been addressed yet? any update on this one @srjhunjhunwalacorp ?

**Comment by srjhunjhunwalacorp** [comment]
> I dont think this issue has been addressed yet? any update on this one @srjhunjhunwalacorp ?

@robertmcleod2 - Appologies for delay I have found a difference in the tag configuration, that explains this outcome and the historian is also giving the same behavior I will finish my check in couple of days and will reach out to you.

**Comment by srjhunjhunwalacorp** [comment]
We are working on it - by next week you will response
